plugins/alphabet-sort/src/contact-card.component.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react'
+import {
+  ContactCardRenderer,
+  ContactCardRendererProps
+} from '@serlo/editor-plugin-alphabet-sort-renderer'
+
+export class ContactCard extends React.Component<ContactCardProps> {
```

Comment:
do you need this component? Seems like it doesn't do much (it just passes stuff down). So I'd rather use that what you are currently doing directly where you use `ContactCard`